src/pages/workspace/accounting/PolicyAccountingPage.tsx

```diff
@@ -105,12 +104,21 @@ function PolicyAccountingPage({policy, connectionSyncProgress}: PolicyAccounting
     const styles = useThemeStyles();
     const {translate} = useLocalize();
     const {isOffline} = useNetwork();
+    const {canUseXeroIntegration} = usePermissions();
     const {isSmallScreenWidth, windowWidth} = useWindowDimensions();
     const [threeDotsMenuPosition, setThreeDotsMenuPosition] = useState<AnchorPosition>({horizontal: 0, vertical: 0});
     const [isDisconnectModalOpen, setIsDisconnectModalOpen] = useState(false);
     const threeDotsMenuContainerRef = useRef<View>(null);
 
     const isSyncInProgress = !!connectionSyncProgress?.stageInProgress && connectionSyncProgress.stageInProgress !== CONST.POLICY.CONNECTIONS.SYNC_STAGE_NAME.JOB_DONE;
+
+    const accountingIntegrations = Object.values(CONST.POLICY.CONNECTIONS.NAME).filter((name) => {
+        if (name === CONST.POLICY.CONNECTIONS.NAME.XERO && !canUseXeroIntegration) {
+            return false;
+        }
+
+        return true;
```

Comment:
Couldn't this be shortened to being

```suggestion
        return !(name === CONST.POLICY.CONNECTIONS.NAME.XERO && !canUseXeroIntegration);
```

Reply:
Nice catch, I've updated code!